repository-data/development/src/main/resources/hcm-content/content/documents/lks/search-bank.yaml

```diff
@@ -122,3 +122,121 @@
         jcr:encoding: UTF-8
         jcr:lastModified: 2021-05-11T12:32:18.067+01:00
         jcr:mimeType: application/vnd.openxmlformats-officedocument.wordprocessingml.document
+  /search-search-bank:
```

Comment:
Could we reload `/content/documents/lks/search-bank` ?

Reply:
Added